primitives/Cargo.toml

```diff
@@ -11,7 +15,7 @@ codec = { package = "parity-scale-codec", version = "3.6.1", default-features =
     "max-encoded-len",
 ] }
 crc = "2.1.0"
-geohash = { tag = "polkadot-v1.0.0", git = "https://github.com/encointer/geohash" }
+geohash = { package = "substrate-geohash", version = "0.13.0" }
```

Comment:
Interesting naming, I would not have thought of that, but I think it is a great fit.